test/functional/test_framework/blocktools.py

```diff
@@ -128,8 +128,8 @@ def add_witness_commitment(block, nonce=0):
 def script_BIP34_coinbase_height(height):
     if height <= 16:
         res = CScriptOp.encode_op_n(height)
-        # Append dummy to increase scriptSig size above 2 (see bad-cb-length consensus rule)
-        return CScript([res, OP_1])
```

Comment:
In commit "Use OP_0 for BIP34 padding in signet and tests" (75342f7e19cbbae083c2a3561f2c2cefd4e2968a)

Do we know any reason why this code was using OP_1 instead of OP_0 previously? Any possible advantages to using OP_1? It looks like this code was introduced in #16363

Reply:
I don't know. @maflcko?